### ISSUE TITLE
Disallow empty lines in object curly, array bracket, or function params

### ISSUE THREAD

**Issue by golopot** [eslint/eslint#11831]
**What rule do you want to change?**
Maybe `object-curly-newline`, `array-bracket-newline`, `function-param-newline`
**Does this change cause the rule to produce more or fewer warnings?**
more
**How will the change be implemented? (New option, new default behavior, etc.)?**
A new option like `{ "forbidEmptyLine": true }` in each of the above rules.
**Please provide some example code that this change will affect:**

```js
// Warns about empty lines
import {

  a,
} from 'cat'

const purr = [
  b,

]

function meow(

  c,
) {}

```
**What does the rule currently do for this code?**
Nothing.
**What will the rule do after it's changed?**
Empty lines are reported.
**Are you willing to submit a pull request to implement this change?**
No.

There are already some rules that disallow empty lines: `no-multiple-empty-lines`, `padding-line-between-statements`, `padded-blocks`, `lines-between-class-members`. But none of them can handle the aforementioned cases.
